packages/rest-typings/src/v1/oauthapps/OAuthAppsInfoParamsGET.ts

```diff
@@ -0,0 +1,16 @@
+import { ajv } from '../Ajv';
+
+export type OauthAppsInfoParams = { clientId: string };
+
+const oauthAppsInfoParamsSchema = {
+	type: 'object',
+	properties: {
+		clientId: {
+			type: 'string',
+		},
```

Comment:
```suggestion
		clientId: {
			type: 'string',
			minLength: 1,
		},
```
It'd be good to validate that this string isn't empty so that we don't fetch a random app from the DB in case an empty string is provided.

Reply:
Thank you very much for the comment. Adding this validation also fixes one of the type issues that I didn't notice until running the tests through GitHub Workflow.